Dockerfile

```diff
@@ -9,12 +9,12 @@ WORKDIR /app
 
 COPY package.json ./package.json
 COPY package-lock.json ./package-lock.json
+RUN npm ci && npm cache clean --force
```

Comment:
I force cleared the cache to try to shrink the final image, but might be better to remove this if the goal is to have faster dev builds

Reply:
no keeping the layers small is always a good idea